packages/atomic/src/components/search-box-suggestions/suggestions-common.ts

```diff
@@ -0,0 +1,41 @@
+import {SearchBox} from '@coveo/headless';
+import {VNode} from '@stencil/core';
+import {buildCustomEvent} from '../../utils/event-utils';
+import {Bindings} from '../../utils/initialization-utils';
+
+export interface SearchBoxSuggestionElement {
+  value: string;
+  onClick(): void;
```

Comment:
Suggestion: `onSelect` since it's not always going to be a click. eg: touch on mobile, or enter is using a keyboard.